### PR TITLE
Added the tabindex option to match capabilities of ampersand-input-view.

### PR DESCRIPTION
**Pull request by kylefarris**
Title sums it up I think. 

You can refer to this PR for more info:
https://github.com/AmpersandJS/ampersand-input-view/pull/70

BTW, sorry for the later-than-expected PR. Completely forgot to submit this one too...

### CODE REVIEW

File: ampersand-select-view.js [hunk 1]
```diff
@@ -76,6 +76,8 @@ module.exports = View.extend({
         this.startingValue = opts.value;
         this.yieldModel = (opts.yieldModel === false) ? false : true;
 
+        this.tabindex = opts.tabindex || '0';
+
```
Comment (wraithgar) [inline ampersand-select-view.js]: Help for other reviewers: this is one of the two actual code additions that isn't whitespace cleanup.

File: ampersand-select-view.js [hunk 2]
```diff
@@ -119,6 +121,7 @@ module.exports = View.extend({
         if (!this.select) throw new Error('no select found in template');
         if (matches(this.el, 'select')) this.select = this.el;
         if (this.select) this.select.setAttribute('name', this.name);
+        if (this.select) this.select.setAttribute('tabindex', this.tabindex);
```
Comment (wraithgar) [inline ampersand-select-view.js]: Help for other reviewers: this is one of the two actual additions that isn't whitespace cleanup.

**Comment by wraithgar** [comment]
+1 from me
I added notes so anyone else reviewing could quickly discern between the whitespace cleanup and the code that actually represents this PR.

File: README.md [hunk 3]
```diff
@@ -34,13 +32,14 @@ Sets the selected option to that which matches the provided value.  Updates the
 - `name`: the `<select>`'s `name` attribute's value. Used when reporting to parent form
 - `parent`: parent form reference
 - `options`: array/collection of options to render into the select box
-- `[groupOptions]`: use instead of `options` to generate &lt;optgroup&gt; elements within your &lt;select&gt;. If this is set, any values passed in `options` will be ignored and replaced with values coming from `groupOptions`.
+- `[groupOptions]`: use instead of `options` to generate <optgroup> elements within your <select>. If this is set, any values passed in `options` will be ignored and replaced with values coming from `groupOptions`.
```
Comment (cdaringe) [inline README.md]: my .md preview seems to think this is a naughty change.

screen shot 2016-02-17 at 10 27 22 am

File: README.md [hunk 4]
```diff
@@ -50,23 +49,22 @@ Sets the selected option to that which matches the provided value.  Updates the
...
 ##### collection option set
 If using a collection to produce `<select>` `<option>`s, the following may also be specified:
-
 - `[disabledAttribute]`: boolean model attribute to flag disabling of the option node
 - `[idAttribute]`: model attribute to use as the id for the option node.  This will be returned by `SelectView.prototype.value`
 - `[textAttribute]`: model attribute to use as the text of the option node in the select box
 - `[yieldModel]`: [default: `true`] if options is a collection, yields the full model rather than just its `idAttribute` to `.value`
 
-When the collection changes, the view will try and maintain its currently `.value`.  If the corresponding model is removed, the &lt;select&gt; control will default to the 0th index &lt;option&gt; and update its value accordingly.
+When the collection changes, the view will try and maintain its currently `.value`.  If the corresponding model is removed, the <select> control will default to the 0th index <option> and update its value accordingly.
```
Comment (cdaringe) [inline README.md]: hence here too

File: README.md (hunk 3 above)
Comment (wraithgar) [inline README.md]: good catch!

**Comment by kylefarris** [comment]
That is weird... I was questioning my beautifying library when it changed those to the actual > and < characters but assumed it was proper... I'll turn them back to the entity versions in VIM and commit. Good find guys, sorry about all the clean up. I mean, I guess it's not a bad thing... just makes the PR a bit messy.

**Comment by kylefarris** [comment]
So, actually, it really seems like the bigger "issue" was the inconsistent way in which HTML elements were written in the markdown. Sometimes they were written without being wrapped in back ticks, other times they were. I changed all occurrences so that they are wrapped in ticks since I think the pre-formatted style is easier to read and probably more "proper" anyway.

**Comment by cdaringe** [comment]
+1

**Pull request merged**